styles/overrides/_styleguidist.scss

```diff
@@ -3,56 +3,250 @@
+// Styleguidist "components" Wrapper
+// ---
+
+.rsg-components {
+    display: flex;
+    flex: 1 1 auto;
+}
+
+
+// Styleguidist "Search" and "Navigation"
+// ---
+
+.rsg-components__search-nav {
+    z-index: $z2-depth - 1;
+
+    overflow: scroll;
+    flex: 0 0 auto;
+    width: $rsgSearchHeaderWidth;
+    margin-top: -40px;
+    padding: $xlarge-space $medium-space;
+
+    background: #fff;
 
     a {
-        display: inline-flex;
-        justify-content: center;
-        align-items: center;
+        color: $link-color;
+        font-size: $base-font-size;
+        text-decoration: none;
+    }
+
+    ul {
+        padding: 0;
+    }
+
+    li {
+        list-style: none;
+    }
+
+    input {
+        width: 100%;
+        padding: $small-space;
+        border: 1px solid $border-color;
+
+        border-radius: 2px;
+        background-color: $white;
+        outline: none;
+
+        transition: all .15s ease-in-out;
+
+        &:focus {
+            box-shadow: $small-box-shadow;
+        }
+    }
+
+    .rsg-component-filter {
+        margin-bottom: $base-space;
+    }
+
+    [class*="rsg--list-"] .rsg-component-filter {
+        display: none;
     }
 }
 
-.ReactStyleguidist-Markdown__h4 {
-    margin-top: $large-space;
+@include for-phone-only {
+    .rsg-components__search-nav {
+        display: none;
+    }
 }
 
-.ReactStyleguidist-Markdown__th,
-.ReactStyleguidist-Markdown__td {
-    border: none !important; // 1
+
+// List of Links (sidebar nav links)
+// ---
+
+.rsg-component-list .rsg-component-list {
+    padding-bottom: $base-space;
 }
 
-.ReactStyleguidist-Markdown__table {
-    color: $neutral-50;
+
+// Styleguidist Components, Main Content Area
+// ---
+
+.rsg-components__component-section {
+    flex: 1 1 auto;
+    max-width: 100%;
+    padding-right: $xlarge-space;
+    padding-left: $xlarge-space;
+
+    [class*="rsg--heading"],
+    [class*="rsg--heading"] a {
+        font-family: $brand-font-family;
+        font-weight: 700;
+        letter-spacing: -.05rem;
+        text-decoration: none;
+    }
+
+    [class*="rsg--heading1-"] {
+        margin-top: $massive-space;
+    }
+
+    [class*="rsg--preview"] {
+        background: $neutral-10;
+    }
+
+    [class*="rsg--pre"] {
+        overflow: scroll;
+        margin-bottom: $xlarge-space;
+    }
+
+    [class*="rsg--table-"] {
+        background: transparent;
+    }
+
+    [class*="rsg--tableHead-"],
+    [class*="rsg--thead-"],
+    [class*="rsg--tr-"],
+    [class*="rsg--table-"] tr {
+        border-bottom: 1px solid rgba(0,0,0,.03);
+    }
+
+    [class*="rsg--cellHeading-"],
+    [class*="rsg--cell-"],
+    [class*="rsg--th-"],
+    [class*="rsg--td-"], {
+        border: none !important; // 1
+
+        color: $neutral-50;
+        font-size: $small-font-size;
+
+        [class*="rsg--para-"] {
+            color: $neutral-50;
+            font-size: $small-font-size;
+        }
+    }
+
+    [class*="rsg--cellHeading-"] {
+        font-size: 10px;
```

Comment:
can we use `$small-font-size` which is 12px instead? if not, let's add a variable on top like `cell-heading-size: 10px` and use it here

Reply:
`cell-heading-size` is a bit presumptuous, I think. I've gone with `$tiny-font-size` to stick with the current theme of font size variables.